lstchain/reco/utils.py

```diff
@@ -21,11 +21,12 @@
 from astropy.time import Time
 from astropy.utils import deprecated
 from ctapipe.coordinates import CameraFrame
+from ctapipe.containers import EventType
 
 from . import disp
 
 __all__ = [
-    'add_delta_t_key'
+    'add_delta_t_key',
```

Comment:
how did this past the tests without that comma??

Reply:
I have no idea! I saw it not because of any misbehaviour, but thanks to PyCharm checks!